preferences.yaml

```diff
@@ -0,0 +1,6 @@
+latest preset:
+  plugin: Cromha Stuff
+  save: Cromha MCcloud
+  type: plugin
```

Comment:
These will have to be reset.

Reply:
Yes.

Reply:
I can't get the `preferences.yaml` be in the .gitignore. Even if I add
```
preferences.yaml
**/preferences.yaml
```
It don't get added.